module/os/linux/zfs/abd_os.c

```diff
@@ -598,7 +598,7 @@ abd_free_chunks(abd_t *abd)
 
 	abd_for_each_sg(abd, sg, n, i) {
 		struct page *p = nth_page(sg_page(sg), 0);
-		umem_free(p, PAGESIZE);
+		umem_free_aligned(p, PAGESIZE);
```

Comment:
We don't need to make this change to the Linux specific files.
Unless the clarification on what type of memory being freed is usefull.

Reply:
yeah, in userland, it only compiles linux files. I'm... not sure.. if that is the way, but it works for now.

Reply:
It is also used on FreeBSD. I am not sure offhand if it is compiled for the FreeBSD kernel offhand.

Reply:
Right, the user space code here isn't Linux specific, it's just hasn't yet been moved to a common location.  If someone wants to tackle that in a different PR I'm all for it.

Reply:
@ryao I didn't change the argument to PAGESIZE that is already in the code.
Did I even need to change these? The second argument isn't even used in umem_free or umem_free_aligned.
Anyway doesn't matter I'm changing them.

Reply:
It is supposed to be the size of the allocation. Here, PAGESIZE is correct.